stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/StagGenerator.java

```diff
@@ -299,8 +299,14 @@ private TypeSpec getAdapterFactorySpec() {
                 .addParameter(Gson.class, "gson")
                 .addParameter(ParameterizedTypeName.get(ClassName.get(TypeToken.class), genericTypeName),
                               "type")
+                .addStatement("setOrThrow(gson)")
```

Comment:
There should be a license header on every file, so I will add it where missed. Good catch.